.golangci.yml

```diff
@@ -214,8 +127,15 @@ issues:
   # Default value for this option is true.
   exclude-use-default: true
 
-  # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
-  max-per-linter: 0
```

Comment:
why remove this?

Reply:
The field is not supportted by the latest golangci-lint, it will report error.

Reply:
It's an dup of `max-issues-per-linter: 0` set above.